onebusaway-android/src/main/java/org/onebusaway/android/io/elements/ObaRegionElement.java

```diff
@@ -256,6 +305,7 @@ public String toString() {
                 ", obaBaseUrl='" + obaBaseUrl + '\'' +
                 ", siriBaseUrl='" + siriBaseUrl + '\'' +
                 ", bounds=" + Arrays.toString(bounds) +
+                ", open311Servers=" + Arrays.toString(open311Servers) +
```

Comment:
Yes it seems we do not need that character for arrays:
![screen shot 2016-04-06 at 10 10 19 pm](https://cloud.githubusercontent.com/assets/2777974/14338368/8f152be4-fc44-11e5-81e3-62479ac77207.png)

However, I don't think `Arrays.tostring` works correctly. It prints the memory address of an array.